security/libprelude/Portfile

```diff
@@ -3,47 +3,55 @@
 PortSystem      1.0
 
 name            libprelude
-version         1.0.1
-set download_id 241
-revision        5
+version         5.1.1
+revision        1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
ok, done